config/settings/test.yml

```diff
@@ -5,4 +5,9 @@ symws:
     login: 'fake_user'
     password: 'some_password'
 symphony:
-  host: example.com
+  host: example.com 
+folio:
+  # this is a feature flag about group fines
+  # FOLIO we set it to true and because sponsor can pay all fines in their group
+  # TODO: remove after launch
+  sponsor_fines: false
```

Comment:
Can this be set in `config/settings.yml` Otherwise, we get:
```
 undefined method `sponsor_fines' for nil:NilClass
```

when not using the `test` environment.